src/core_ocean/shared/mpas_ocn_mesh.F

```diff
@@ -1044,9 +1044,11 @@ subroutine ocn_meshUpdateFields(domain) !{{{
       end do
       end do
 
+      ! Allocatable array was updated in ocn_init_routines_setup_sign_and_index_fields.  Now the 
+      ! pointer needs to be udpated.
       do n = 1, nCellsAll+1
       do k = 1, maxEdges
-         edgeSignOnCell(k, n) = real(edgeSignOnCellTmp(k, n), RKIND)
+         edgeSignOnCellTmp(k, n) = int(edgeSignOnCell(k, n))
```

Comment:
Might be good to keep the pointers updated in case someone still accesses via the meshPool. The current mesh module is back-compatible with pools just so we could do incremental development, but we will eventually get rid of all the pointers here (and Registry entries) in favor of just allocatable arrays in this module.